analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/ExecutingSqlQueriesBase.cs

```diff
@@ -95,7 +95,22 @@ public abstract class ExecutingSqlQueriesBase<TSyntaxKind, TExpressionSyntax, TI
 
         private readonly MemberDescriptor[] invocationsForFirstArgument =
             {
-                new(KnownType.System_Data_Sqlite_SqliteCommand, "Execute")
+                new(KnownType.System_Data_Sqlite_SqliteCommand, "Execute"),
+                new(KnownType.NHibernate_ISession, "CreateQuery"),
+                new(KnownType.NHibernate_ISession, "CreateSQLQuery"),
+                new(KnownType.NHibernate_ISession, "Delete"),
+                new(KnownType.NHibernate_ISession, "DeleteAsync"),
+                new(KnownType.NHibernate_ISession, "GetNamedQuery"),
+                new(KnownType.NHibernate_Impl_SessionImpl, "CreateQuery"),
+                new(KnownType.NHibernate_Impl_SessionImpl, "CreateSQLQuery"),
+                new(KnownType.NHibernate_Impl_SessionImpl, "Delete"),
+                new(KnownType.NHibernate_Impl_SessionImpl, "DeleteAsync"),
+                new(KnownType.NHibernate_Impl_SessionImpl, "GetNamedQuery"),
+                new(KnownType.NHibernate_Impl_SessionImpl, "GetNamedSQLQuery"),
```

Comment:
This is not needed, it's tracked from the `AbstractSessionImp`. Also KnownType can be removed.
```suggestion
```